spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/config/GatewayAutoConfiguration.java

```diff
@@ -212,15 +182,15 @@ public RouteDefinitionLocator routeDefinitionLocator(List<RouteDefinitionLocator
 
 	@Bean
 	public ConfigurationService gatewayConfigurationService(BeanFactory beanFactory,
-			@Qualifier("webFluxConversionService") ObjectProvider<ConversionService> conversionService,
-			ObjectProvider<Validator> validator) {
+															@Qualifier("webFluxConversionService") ObjectProvider<ConversionService> conversionService,
```

Comment:
Please don't change spacing. It makes it difficult to review

Reply:
ok, I restore it